src/EventStoreCatchUpSubscription.php

```diff
@@ -26,6 +26,4 @@ public function subscriptionName(): string;
     public function start(): void;
```

Comment:
here we need to keep it anyway, because the user has to call this method initially.